### PR TITLE
[full-ci] bump core commit id to run new versions tests

### PR DESCRIPTION
**Pull request by individual-it**
## Description
run tests of https://github.com/owncloud/core/pull/38921

after https://github.com/owncloud/core/pull/38921 is merged this need to be changed to have master branch and the correct commit id in the .env file

## How Has This Been Tested?
:robot:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>localApiTests-apiBugDemonstration-ocis</strong> failed. The build is cancelled...</summary>

**Comment by phil-davis** [comment]
@individual-it there days you can put `full-ci` in the PR title, and all the test pipelines will run to completion. That is handy when you need to see all the things that fail.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>localApiTests-apiBugDemonstration-ocis</strong> failed. The build is cancelled...</summary>

**Comment by phil-davis** [comment]
IMO you need to push a change or rebase. Just restarting the drone build does not make it fetch the updated PR title.

**Comment by individual-it** [comment]
that failing test does not make any sense in the first place, it does not use `Shares/` to access the received share and also the original test https://github.com/owncloud/core/blob/b8e8d740b24e18da4e951b99302d2034f2e7215d/tests/acceptance/features/apiVersions/fileVersionsSharingToShares.feature#L43 is not listed in expected to fail, so should pass

**Comment by phil-davis** [comment]
Those "local API" "bug demo" scenarios don't seem to try to use the `Shares` folder at all. It's a long time since anyone looked at them. They need to be reviewed and sorted out.

**Comment by phil-davis** [comment]
But I think the cause of the fail is https://github.com/owncloud/core/pull/38921#pullrequestreview-699133898

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>localApiTests-apiBugDemonstration-ocis</strong> failed. The build is cancelled...</summary>

File: tests/acceptance/expected-failures-API-on-OWNCLOUD-storage.md [hunk 1]
```diff
@@ -274,6 +260,8 @@ _ocdav: return checksum in upload response for chunked upload_
 -   [apiWebdavLocks2/resharedSharesToShares.feature:59](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavLocks2/resharedSharesToShares.feature#L59)
 -   [apiWebdavLocks2/resharedSharesToShares.feature:60](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavLocks2/resharedSharesToShares.feature#L60)
 -   [apiWebdavLocks2/resharedSharesToShares.feature:61](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavLocks2/resharedSharesToShares.feature#L61)
+-   [apiWebdavLocks2/resharedSharesToShares.feature:78](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavLocks2/resharedSharesToShares.feature#L78)
+-   [apiWebdavLocks2/resharedSharesToShares.feature:79](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavLocks2/resharedSharesToShares.feature#L79)
```
Comment (individual-it) [inline tests/acceptance/expected-failures-API-on-OWNCLOUD-storage.md:264]: I have no idea how those tests have passed before as locking is not implemented in ocis, maybe @saw-jan does know

**Comment by phil-davis** [comment]
Note: @kiranparajuli589 has #2264 almost ready - that will adjust a lot of stuff.

**Comment by individual-it** [comment]
#2264 bumps the we commit id, here we bump the core id

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/ocis/5841/37/7
```
runsh: There were no unexpected failures.
runsh: Total unexpected passed scenarios throughout the test run:
apiVersions/fileVersions.feature:420
apiVersions/fileVersions.feature:438
apiVersions/fileVersionsSharingToShares.feature:301
```

https://drone.owncloud.com/owncloud/ocis/5841/38/6
```
runsh: There were no unexpected failures.
runsh: Total unexpected passed scenarios throughout the test run:
apiVersions/fileVersions.feature:420
apiVersions/fileVersions.feature:438
apiVersions/fileVersionsSharingToShares.feature:301
```

**Comment by individual-it** [comment]
the only reason why they pass is because they did not run :-( but why?

**Comment by individual-it** [comment]
I've copied the wrong commit id from master, lets see now

**Review by phil-davis — APPROVED** [review]

**Pull request merged**